Tests/ApodiniTests/WebSocketInterfaceExporterTests.swift

```diff
@@ -297,17 +297,24 @@ class WebSocketInterfaceExporterTests: XCTApodiniTest {
         let errorForwardingExporter = ErrorForwardingInterfaceExporter {
             forwardedError = $0
         }
+        print("register exporter")
```

Comment:
As I'm just randomly checking over this PR (to ensure this doesn't break anything on my side), might remove those debug print statements before merging 🙃

Reply:
yeah i forgot to add a note on here but the PR is currently on-hold bc there's some test failures caused by the WebSocketIE which i'm also trying to fix via this PR, the print stmt you found was part of that there's probably some more